be/src/exec/aggregator.cpp

```diff
@@ -413,6 +414,23 @@ Status Aggregator::prepare(RuntimeState* state, ObjectPool* pool, RuntimeProfile
             Expr* expr = nullptr;
             ExprContext* ctx = nullptr;
             RETURN_IF_ERROR(Expr::create_tree_from_thrift(_pool, desc.nodes, nullptr, &node_idx, &expr, &ctx, state));
+
+            if (state->is_jit_enabled()) {
```

Comment:
why not use `Status Expr::create_expr_tree(`? 
or move JIT code to `create_tree_from_thrift`?

Reply:
as these code are duplicated with those in `create_expr_tree`

Reply:
Moving the JIT code to ﻿`create_tree_from_thrift` can be complex, as determining if an expr is compile-able requires traversing down the child expr. This process would be more manageable after ﻿`create_tree_from_thrift` is completed.
The original code invokes ﻿`create_tree_from_thrift`, which I haven't modified. It seems that ﻿`create_expr_tree` could be a viable alternative.